Valet.podspec

```diff
@@ -1,12 +1,13 @@
 Pod::Spec.new do |s|
   s.name     = 'Valet'
-  s.version  = '3.2.3'
+  s.version  = '3.2.4'
+  s.cocoapods_version = '>= 1.7.0'
```

Comment:
What does this line entail? Is this change backwards compatible?

Reply:
It is I was just about to comment to that. It should be backwards compatible all the way back to CocoaPods 1.4.0 when the `swift_version` (singular) DSL was first introduced.

Reply:
Awesome!

Reply:
You can remove this line btw. Once this PR lands prior versions all the way down to 1.4.0 will pick Swift 5.0 as their version.